plugins/exec/README.md

```diff
@@ -18,3 +17,23 @@ The name is used as a prefix for the measurements.
 The interval is used to determine how often a particular command should be run. Each
 time the exec plugin runs, it will only run a particular command if it has been at least
 `interval` seconds since the exec plugin last ran the command.
+
+
+# Sample
+
+Let's say that we have a command which gives the following output:
+```json
+{
+    "a": 0.5,
+    "b": {
+        "c": "some text",
+        "d": 0.1
+    }
+}
+```
+
+The collected metrics will be:
+```
+exec_catter_a value=0.5
```

Comment:
explain where `catter` comes from as well, I'm assuming this is from `name`?

Also, while you're here, can you remove the `interval` section of the README, this is redundant since it's the same for every plugin